src/community/oseo/web-oseo/pom.xml

```diff
@@ -61,13 +61,6 @@
       <classifier>tests</classifier>
       <scope>test</scope>
     </dependency>
-    <dependency>
```

Comment:
This part looks unrelated and possibly undesired.

Reply:
when you look ~10 lines up you can see the duplicate declaration: https://github.com/geoserver/geoserver/blob/0ef63e17696b7b4c278e78b3138968101c16d0f9/src/community/oseo/web-oseo/pom.xml#L43-L49